src/routes/oauthGuilds.js

```diff
@@ -12,6 +12,13 @@ module.exports = class extends Route {
 
 	async post(request, response) {
 		const botGuild = this.client.guilds.get(request.body.id);
+		const member = await botGuild.members.fetch(request.auth.scope[0]);
```

Comment:
If the member does not exist, this will throw instead of sending a notFound message.